### ISSUE TITLE
How can I do "If loss = 7 in THE LAST 10 trade. Wait 300 sec. "

### ISSUE THREAD

**Issue by freedayz04** [binary-com/binary-bot#2502]
[Over4Bot By Freedayz04.zip](https://github.com/binary-com/binary-bot/files/4261291/Over4Bot.By.Freedayz04.zip)
Plz Help me.
Best regards.

**Comment by JenggotSantuy** [comment]
just input if you need help then you should also include bots so that those who want to help can see and revise them 🙏

**Comment by freedayz04** [comment]
> just input if you need help then you should also include bots so that those who want to help can see and revise them 🙏

Thank you . I have updated bot.

**Comment by JenggotSantuy** [comment]
does that mean your problem is resolved?

**Comment by freedayz04** [comment]
> does that mean your problem is resolved?

No . I mean I've put this bot for the post.

**Comment by JenggotSantuy** [comment]
before talking about the revision 
I want to give your opinion bot very risky if the order without analysis the results are not necessarily sometimes the win is greater sometimes the loss is greater so there is no benchmark 
Have you tried your back to under 4? I'm waiting for your opinion

**Comment by JenggotSantuy** [comment]
Hi @freedayz04 
I have given advice but you don't want to respond

please try according to your wishes
[Revisi300Sec.zip](https://github.com/binary-com/binary-bot/files/4270269/Revisi300Sec.zip)
